internal/acceptance/entitlements_test.go

```diff
@@ -95,3 +103,122 @@ func TestAccServicePrincipalEntitlementsResourceOnAws(t *testing.T) {
 		}`,
 	})
 }
+
+type entitlement struct {
+	name  string
+	value bool
+}
+
+func (e entitlement) String() string {
+	return fmt.Sprintf("%s = %t", e.name, e.value)
+}
+
+func entitlementsStepBuilder(t *testing.T, c **httpclient.ApiClient, groupName string) func(entitlements []entitlement) step {
+	return func(entitlements []entitlement) step {
+		entitlementsBuf := strings.Builder{}
+		for _, entitlement := range entitlements {
+			entitlementsBuf.WriteString(fmt.Sprintf("%s\n", entitlement.String()))
+		}
+		return step{
+			Template: fmt.Sprintf(`
+			data "databricks_group" "example" {
+				display_name = "%s"
+			}
+			
+			resource "databricks_entitlements" "entitlements_users" {
+				group_id              = data.databricks_group.example.id
+				%s
+			}
+		`, groupName, entitlementsBuf.String()),
+			Check: func(s *terraform.State) error {
+				groupId := s.RootModule().Resources["data.databricks_group.example"].Primary.ID
+				var res iam.Group
+				ctx := context.Background()
+				err := (*c).Do(ctx, "GET", fmt.Sprintf("/api/2.0/preview/scim/v2/Groups/%s?attributes=entitlements", groupId),
```

Comment:
we should expose attributes as a parameter in the SDK to clean this up later

Reply:
Yes, absolutely.